jina/orchestrate/flow/base.py

```diff
@@ -147,11 +147,15 @@ def __init__(
         deployments_addresses: Optional[str] = '{}',
         deployments_disable_reduce: Optional[str] = '[]',
         description: Optional[str] = None,
+        disable_auto_volume: Optional[bool] = False,
```

Comment:
gateway parser now has container capability and this will be fixed once we separate flow and gateway API here: https://github.com/jina-ai/jina/issues/5215